src/earn/EarnCta.tsx

```diff
@@ -1,36 +1,67 @@
+import BigNumber from 'bignumber.js'
 import React from 'react'
 import { useTranslation } from 'react-i18next'
 import { StyleSheet, Text, View } from 'react-native'
 import { EarnEvents } from 'src/analytics/Events'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import TokenIcon, { IconSize } from 'src/components/TokenIcon'
 import Touchable from 'src/components/Touchable'
-import EarnAave from 'src/icons/EarnAave'
+import { PROVIDER_ID } from 'src/earn/constants'
+import { useAavePoolInfo } from 'src/earn/hooks'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import Colors from 'src/styles/colors'
 import { typeScale } from 'src/styles/fonts'
 import { Spacing } from 'src/styles/styles'
+import { useTokenInfo } from 'src/tokens/hooks'
+import Logger from 'src/utils/Logger'
+
+const TAG = 'earn/EarnCta'
 
 export default function EarnCta({ depositTokenId }: { depositTokenId: string }) {
   const { t } = useTranslation()
+  const depositToken = useTokenInfo(depositTokenId)
+  const asyncPoolInfo = useAavePoolInfo({ depositTokenId })
+
+  if (!depositToken) {
+    // should never happen
+    Logger.error(TAG, `No deposit token found: ${depositTokenId}`)
+    return null
+  }
+
+  const apyDisplay = asyncPoolInfo.result
+    ? new BigNumber(asyncPoolInfo.result.apy).multipliedBy(100).toFixed(2)
```

Comment:
annoying unhelpful nit: multiplying an APY by 100 is an odd thing to do (since "APY" implies it's already a percentage and why would we multiple an percentage by 100?).

i know the naming predates this PR, but similar naming issues have caused small bugs in the swap experience. naming something "percentage" when it's really a proportion, or vice versa.

Reply:
I've always thought multiplying by 100 is a display thing, and the value is always a proportion. (like how excel / google sheets display percentages, the underlying value is always a proportion). This makes it easier for calculations too on wherever we use it. If this has caused issues in the past, we can change it to something else in a follow up issue.

Reply:
That makes sense.  The issues we've ran into our with 3rd parties.  E.g., `slippagePercentage` here https://0x.org/docs/0x-swap-api/api-references/get-swap-v1-quote.

The convention that the wallet always stores a proportion sounds good.